spec/controllers/service_requests/post_feedback_spec.rb

```diff
@@ -1,85 +0,0 @@
-# Copyright © 2011 MUSC Foundation for Research Development
```

Comment:
This method hits an external service and I attempted to VCR (https://github.com/vcr/vcr), which hits the service once and then puts the details of the request into a yml file. The problem was that Travis isn't configured with that, so while it was passing locally, it was failing on CI. I tried playing around with it, but couldn't get it to pass. To test this, we will definitely need to stub it out somehow, because we don't want to ping Redcap everytime we run our specs.